backend/src/main/resources/database/db.changelog-1.0.xml

```diff
@@ -643,4 +643,30 @@
 			baseTableName="ASSOCIATION_ULTICAL_USERS" referencedColumnNames="id"
 			onDelete="CASCADE" />
 	</changeSet>
+	<changeSet author="bas" id="bas41">
+		<tagDatabase tag="v1.1" />
+	</changeSet>
+	<changeSet author="bas" id="bas42">
+		<addColumn tableName="TOURNAMENT_FORMAT">
+			<column name="association" type="BIGINT"></column>
+		</addColumn>
+		<addForeignKeyConstraint constraintName="fk_tournament_format_association"
+			referencedTableName="ASSOCIATION" baseColumnNames="association"
+			baseTableName="TOURNAMENT_FORMAT" referencedColumnNames="id" />
+		<dropColumn tableName="TOURNAMENT_FORMAT" columnName="dfv_official" />
+	</changeSet>
+	<changeSet author="bas" id="bas43">
+		<addColumn tableName="DIVISION_REGISTRATION">
+			<column name="division_identifier" type="VARCHAR(50)"
+				defaultValue="">
```

Comment:
I just wanted to make sure the notNull constraint doesn't complain,when inserting nothing...

But how do I change a pull request?